### PR TITLE
Added tests for search on map page.

### PR DESCRIPTION
**Pull request by CaCO3ua**

### CODE REVIEW

**Review by JugerJuger — APPROVED** [review]

**Review by MelnykNikita — APPROVED** [review]

File: src/test/java/rv/atqc/gttesting/SearchOnMapPage/SearchOnMapPageChrome.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	}
+	
+	@AfterClass
+	public void quitDriver(){
+	      driver.quit();
+	}
+	
+	@Test
+	public void searchForExistingPlace(){
+		Assert.assertTrue(placesPage.lookForItem("Basiv Kyt").getTextSearchResults().isDisplayed());
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/SearchOnMapPage/SearchOnMapPageChrome.java]: move to const or variable

File: src/test/java/rv/atqc/gttesting/SearchOnMapPage/SearchOnMapPageChrome.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	}
+	
+	@Test
+	public void searchForExistingPlace(){
+		Assert.assertTrue(placesPage.lookForItem("Basiv Kyt").getTextSearchResults().isDisplayed());    
+	}
+	
+	@Test
+	public void searchForExistingTrack(){
+		Assert.assertTrue(placesPage.lookForItem("AWESOME").getTextSearchResults().isDisplayed());
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/SearchOnMapPage/SearchOnMapPageChrome.java]: move to const or variable

File: src/test/java/rv/atqc/gttesting/SearchOnMapPage/SearchOnMapPageChrome.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	}
+	
+	@Test
+	public void searchForExistingTrack(){
+		Assert.assertTrue(placesPage.lookForItem("AWESOME").getTextSearchResults().isDisplayed());    
+	}
+	
+    @Test
+    public void shortSeacthRequest(){
+		Assert.assertTrue(placesPage.lookForItem("12").getFailedRequestField().isDisplayed());
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/SearchOnMapPage/SearchOnMapPageChrome.java]: move to const or variable

File: src/test/java/rv/atqc/gttesting/SearchOnMapPage/SearchOnMapPageChrome.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    }
+	
+    @Test
+    public void searchForNonExistingItem(){
+		Assert.assertEquals(placesPage.lookForItem("bla-bla"). getTextSearchResults().getText(), placesPage.getNoItemMessage());
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/SearchOnMapPage/SearchOnMapPageChrome.java]: move to const or variable

File: src/test/java/rv/atqc/gttesting/SearchOnMapPage/SearchOnMapPageChrome.java [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    }
+	
+  	@Test
+	public void checkSearchInputFieldSecurity(){
+  		Assert.assertEquals(placesPage.lookForItem("<script>alert( 'Hello, World!' );</script>"). getTextSearchResults().getText(), placesPage.getNoItemMessage());
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/SearchOnMapPage/SearchOnMapPageChrome.java]: move to const or variable

**Review by dmatsev — APPROVED** [review]
with comments

**Pull request merged**